include/cereal/archives/json.hpp

```diff
@@ -731,7 +731,7 @@ namespace cereal
       //! Convert a string to an unsigned long long
       void stringToNumber( std::string const & str, unsigned long long & val ) { val = std::stoull( str ); }
       //! Convert a string to a long double
-      void stringToNumber( std::string const & str, long double & val ) { val = std::stold( str ); }
+      void stringToNumber( std::string const & str, long double & val ) { val = std::stold( std::wstring{str.begin(), str.end()} ); }
 
```

Comment:
just one question -- this still works correctly for linux builds right?

Reply:
I remember that last year I tested kpe-core compilation using our Ubuntu Docker image.
But just to be safe I tested again using the following cmake flags:
```
  -DCMAKE_BUILD_TYPE=Release
  -DKPSR_WITH_ZMQ=false
  -DKPSR_TEST_PERFORMANCE=true -DKPSR_WITH_SOCKET=true -DKPSR_WITH_CODE_METRICS=true
  -DKPSR_ENABLE_EXAMPLES=true -DKPSR_ENABLE_TESTS=true
```
And it worked fine